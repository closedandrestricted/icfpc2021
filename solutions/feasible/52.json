{"vertices":[[11,62],[54,45],[32,35],[33,72],[31,47],[9,40],[56,59],[14,64],[31,75],[35,75],[33,45],[35,54],[14,65],[11,39]]}