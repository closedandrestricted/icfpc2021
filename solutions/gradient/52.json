{"vertices":[[74,25],[52,26],[37,45],[50,25],[61,47],[19,30],[33,45],[37,46],[41,66],[44,69],[6,44],[59,54],[20,71],[29,46]]}