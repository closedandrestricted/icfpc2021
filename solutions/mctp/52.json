{"vertices":[[33,14],[70,28],[53,11],[23,36],[48,34],[76,13],[35,59],[72,37],[64,56],[61,58],[8,61],[49,41],[49,78],[23,79]]}