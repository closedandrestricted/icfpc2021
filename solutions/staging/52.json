{"vertices":[[56,65],[69,36],[62,13],[32,61],[47,42],[44,28],[19,38],[23,40],[30,59],[29,63],[11,64],[8,58],[6,67],[29,79]]}