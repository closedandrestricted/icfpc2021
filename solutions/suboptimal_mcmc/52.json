{"vertices":[[20,26],[50,32],[62,11],[8,47],[33,47],[72,32],[19,71],[57,51],[44,35],[45,72],[17,44],[34,54],[23,80],[3,63]]}