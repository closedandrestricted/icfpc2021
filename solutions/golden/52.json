{"vertices":[[0,44],[16,31],[4,52],[3,68],[23,53],[3,75],[26,81],[27,77],[11,64],[48,66],[38,56],[30,54],[66,51],[46,34]]}